src/theme.scss

```diff
@@ -41,6 +42,14 @@ mat-icon[mat-card-avatar] {
   line-height: 40px;
 }
 
+// Fix Icon sizes used in table avatars
+mat-icon[mat-table-avatar] {
```

Comment:
we should not need this one anymore, or if we do, it should be with the other icon size utility classes with the same syntax